dts/bindings/mtd/st,stm32-eeprom.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2019 Kwon Tae-young <tykwon@m2i.co.kr>
+# SPDX-License-Identifier: Apache-2.0
+
+include: eeprom-base.yaml
+
+title: STM32 EEPROM devices
+
+description: |
+    STM32 compatible EEPROM
```

Comment:
Perhaps mention that is the on-chip EEPROM here?

Reply:
Done